faqs.mdx

```diff
@@ -1,5 +1,53 @@
 ---
 title: 'FAQs'
-description: 'Documentation for Faqs'
+description: 'Answers to your most common questions about Amplication usage, management, and implementation'
 icon: 'question'
----
\ No newline at end of file
+---
+
+Here are some of the most common questions we receive about Amplication. If you don't find the answer you're looking for, please reach out to us on [Discord](https://amplication.com/discord) or [contact our team](https://amplication.com/contact-us) directly.
+
+<AccordionGroup>
+<Accordion title="What's the difference between Blueprints and Plugins?">
+Blueprints are like architectural drawings that define what you want to build - they specify the type of resource, its properties, and relationships with other blueprints. Think of them as templates that standardize what you build across your organization.
```

Comment:
I would not use "template" here because it may create more confusion
```suggestion
Blueprints are like architectural drawings that define what you want to build - they specify the type of resource, its properties, and relationships with other blueprints. Think of them as basic models that standardize what you build across your organization.
```